tests/antithesis/avalanchego/main.go

```diff
@@ -409,9 +425,26 @@ func (w *workload) issueXChainOperationTx(ctx context.Context) {
 		zap.Duration("duration", time.Since(operationStartTime)),
 	)
 
-	w.confirmXChainTx(ctx, createAssetTx)
+	if err := w.confirmXChainTx(ctx, createAssetTx); err != nil {
+		w.log.Warn("failed to confirm transaction",
+			zap.String("chain", "X"),
+			zap.String("txType", "createAsset"),
+			zap.Error(err),
+		)
+		return
+	}
+
 	w.verifyXChainTxConsumedUTXOs(ctx, createAssetTx)
-	w.confirmXChainTx(ctx, operationTx)
+
+	if err := w.confirmXChainTx(ctx, operationTx); err != nil {
+		w.log.Warn("failed to confirm transaction",
+			zap.String("chain", "X"),
+			zap.String("txType", "operation"),
+			zap.Error(err),
+		)
+		return
+	}
+
 	w.verifyXChainTxConsumedUTXOs(ctx, operationTx)
```

Comment:
should this method be refactored to return an error? It seems to just log an error if one occurs during but then returns nothing. 

I could create a follow up PR if so

Reply:
Because this function doesn't actually perform any modifications, it isn't _required_ to propagate the error up... So that's why I didn't include that in this PR... But it certainly _could_.